From 2e680276f0d8b60d66cb7a9a439a3169dbafca52 Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Sat, 10 Feb 2018 19:44:00 -0500
Subject: [PATCH 02/27] bsd shm

Debians's kFreeBSD has a Linux like userspace, and
libgnuradio-runtime needs the librt link.
---
 gnuradio-runtime/lib/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnuradio-runtime/lib/CMakeLists.txt b/gnuradio-runtime/lib/CMakeLists.txt
index 3da550d37b..7be913eb30 100644
--- a/gnuradio-runtime/lib/CMakeLists.txt
+++ b/gnuradio-runtime/lib/CMakeLists.txt
@@ -153,7 +153,7 @@ IF(HAVE_WINDOWS_H)
 ENDIF(HAVE_WINDOWS_H)
 
 #need to link with librt on ubuntu 11.10 for shm_*
-if(LINUX)
+if((LINUX) OR (CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD"))
     list(APPEND gnuradio_runtime_libs rt)
 endif()
 
-- 
2.11.0

